sklearn/linear_model/tests/test_ridge.py

```diff
@@ -661,6 +662,19 @@ def _test_ridge_cv(filter_):
     assert type(ridge_cv.intercept_) == np.float64
 
 
+@pytest.mark.parametrize(
+    "ridge, make_dataset",
+    [(RidgeCV(), make_regression),
+     (RidgeClassifierCV(), make_classification)]
+)
+def test_ridge_gcv_cv_values_not_stored(ridge, make_dataset):
```

Comment:
note that the previous implementation would have passed this test because the cv values were stored in a local variable during fit

Reply:
True and we don't have a real good test to detect the enhancement actually.